src/components/LayoutTree.tsx

```diff
@@ -878,6 +893,9 @@ const LayoutTree = () => {
   // Subtract singleLineHeight since we can assume that the last rendered thought is within the viewport. (It would be more accurate to use its exact rendered height, but it just means that there may be slightly more space at the bottom, which is not a problem. The scroll position is only forced to change when there is not enough space.)
   const spaceBelow = viewportHeight - navAndFooterHeight - CONTENT_PADDING_BOTTOM - singleLineHeight
 
+  // Create a ref for each node. This helps to avoid the `findDOMNode is deprecated` warning.
+  //const nodeRef = createRef<HTMLDivElement>()
```

Comment:
Will this remain unused, or is this a WIP?

If `nodeRef` is not needed, it would be better to remove it instead of leaving it commented.